indigo_app/templates/indigo_api/_task_cards.html

```diff
@@ -65,26 +65,48 @@ <h6 class="text-center mb-4">
                         </button>
                         <div class="dropdown-menu" aria-labelledby="unassignDropdownMenuButton">
                           <button class="dropdown-item mb-1" type="submit" data-confirm="Are you sure you want to unassign this task?" formaction="{% url 'unassign_task' place=place.place_code pk=task.pk %}?next={{ request.get_full_path|urlencode }}#task-{{ task.id }}">Unassign</button>
-                          {% for potential_assignee in task.potential_assignees %}
-                          <button class="dropdown-item mb-1" type="submit" name="user_id" value="{{potential_assignee.id }}">{% user_display potential_assignee %}</button>
-                          {% endfor %}
+                          {% if task.state == 'open' %}
+                            {% for potential_assignee in task.potential_assignees %}
+                            <button class="dropdown-item mb-1" type="submit" name="user_id" value="{{ potential_assignee.id }}">{% user_display potential_assignee %}</button>
+                            {% endfor %}
+                          {% elif task.state == 'pending_review' %}
+                            {% for potential_assignee in task.potential_reviewers %}
+                            <button class="dropdown-item mb-1" type="submit" name="user_id" value="{{ potential_assignee.id }}">{% user_display potential_assignee %}</button>
+                            {% endfor %}
+                          {% endif %}
                         </div>
                       </div>
                     </form>
                   {% endif %}
 
                 </div>
-              {% elif task.state == 'open' or task.state == 'pending_review' %}
+              {% elif task.state == 'open' %}
```

Comment:
These two blocks are identical except for `potential_assignees` vs `potential_reviewers`. Rather than duplicating so much html, can we not decide on which one to use at the last minute? Alternatively, isn't a `potential_assignees` always `potential_reviewers` for tasks that are waiting to be reviewed?

Reply:
quite right – I've changed it so that `potential_assignees` points to the right thing on each task (or at least that was the aim)